rocksdb_admin/admin_handler.cpp

```diff
@@ -57,6 +57,9 @@ DEFINE_string(shard_config_path, "",
 DEFINE_bool(rocksdb_allow_overlapping_keys, false,
             "Allow overlapping keys in sst bulk load");
 
+DEFINE_string(allow_overlapping_keys_segments, "",
+              "comma separated list of segments supporting overlapping keys");
```

Comment:
Can we add "it takes priority over rocksdb_allow_overlapping_keys"

Reply:
I'm not sure thats accurate. If FLAGS_rocksdb_allow_overlapping_keys = true and a segment doesn't exist in allow_overlapping_keys_segments, we still allow overlapping keys for the segment